src/DeepCopy/DeepCopy.php

```diff
@@ -40,35 +39,43 @@
      */
     private $typeFilters = [];
 
-    private $skipUncloneable = false;
+    private $skipUncloneable;
     private $useCloneMethod;
 
     /**
-     * @param bool $useCloneMethod If set to true, when an object implements the __clone() function, it will be used
-     *                             instead of the regular deep cloning.
+     * @param bool $useCloneMethod           If set to true, when an object implements the __clone() function, it will
+     *                                       be used instead of the regular deep cloning.
+     * @param bool $skipUncloneable          If enabled, will not throw an exception when coming across an uncloneable
+     *                                       property.
+     * @param Array<Filter, Matcher>         List of filter-matcher pairs
+     * @param Array<TypeFilter, TypeMatcher> List of type filter-matcher pairs
```

Comment:
I don't like very much the use of generics in PHPDoc because there is zero support for them into the language.

Someone could interpret this as a key-value pair, which it isn't here.